python/lsst/ts/xml/data/sal_interfaces/MTAOS/MTAOS_Commands.xml

```diff
@@ -160,4 +160,21 @@
       <Count>1</Count>
     </item>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTAOS</Subsystem>
+    <EFDB_Topic>MTAOS_command_offsetDOF</EFDB_Topic>
+    <Description>Utility command to apply offsets to the degrees of freedom (DOF). Units will match that of the degreeOfFreedom event.</Description>
+    <item>
+      <EFDB_Name>value</EFDB_Name>
+      <Description>Array with the offsets to be applied to the DOF.</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>micron</Units>
```

Comment:
This part how about we use the `unitless` instead and put the following to the description: "Please check the file of  state0inDof.yaml in ts_ofc/policy"?

The benefit is that the user will know the meaning of each DOF.

Otherwise, it would be very confusing because the 50 DOFs have the units of `um` and  `arcsec`.

In addition, because the OFC will accumulate the internal DOF at each iteration, you may want to identify this command will overwrite the current accumulated DOF or accumulate with it? This question applies to the `MTAOS_command_resetOffsetDOF` as well.